pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -696,8 +735,8 @@ def send_notification_email(self, advisories: Dict[str, int], jira_link: str):
               help="The name of an assembly to rebase & build for. e.g. 4.9.1")
 @click.option("--name", metavar="RELEASE_NAME",
               help="release name (e.g. 4.6.42)")
-@click.option("--date", metavar="YYYY-MMM-DD", required=True,
-              help="Expected release date (e.g. 2020-11-25)")
+@click.option("--date", metavar="YYYY-Mon-DD", required=False,
```

Comment:
I think we shouldn't allow users to specify release date with `--data` anymore if it is already defined in releases.yml. Would like to completely remove this option.